source/math/FloatMatrix.ooc

```diff
@@ -288,6 +301,58 @@ operator * (left: FloatMatrix, right: FloatMatrix) -> FloatMatrix {
 result
 }
 
+// <summary>
+// Multiplication of scalar and matrix.
+// </summary>
+// <param name="left">Left matrix in the subtraction.</param>
+// <param name="right">Right matrix in the subtraction.</param>
+// <returns>Difference of left and right matrices.</returns>
```

Comment:
Haha, I think these function descriptions are being thrown away but I shall update it anyway.